app/bundles/FormBundle/Form/Type/FormFieldSliderType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
```

Comment:
Declare strict types for all new classes, please
```suggestion

declare(strict_types=1);

```